firmware/validators.py

```diff
@@ -0,0 +1,36 @@
+import sqlite3
+import os
+from flask import Flask, request, session, g, redirect, url_for, abort, render_template, flash
+import pdb
+
+def validate_company(request_form, request_files):
```

Comment:
Please rename `request_form` to `company` and `request_files` to `company_files` or `files`.